### ISSUE TITLE
Show contributors in repository summary (`.ui.segment.sub-menu`)

### ISSUE THREAD

**Issue by alexanderadam** [go-gitea/gitea#7796]
## Description

It would be nice to see repository size (similar like [`github-repo-size`](https://github.com/harshjv/github-repo-size) does it) and the amount of contributors:

![screenshot](https://user-images.githubusercontent.com/372620/62685854-c2d4d480-b9c3-11e9-8d91-51c263fbb14f.png)

Maybe even the amount of tags could be helpful for some.

**PS:** GitLab does this as well but visually differently:

![Screenshot_20190808_100447](https://user-images.githubusercontent.com/372620/62685950-fc0d4480-b9c3-11e9-94ec-3e7f094b6b25.png)

**PPS:** I thought there was already an open issue for this but I'm unable to find it‽

**Comment by oscarlofwenhamn** [comment]
Agreed, this would make it easier for both users and administrators to find causes of large repositories as well, in order to keep repo sizes down.

**Comment by sapk** [comment]
I will add the repo size in #7833. The PR currently only add it in the settings of repo.
For admin purpose, you allready see the size of every repos in the administration pages

**Comment by alexanderadam** [comment]
> I will add the repo size in #7833

Wow, awesome! Thank you!

**Comment by IzzySoft** [comment]
Looking forward to the contributors number (and list?), too – really miss that!
